Introduce component template inheritance (#462)
github/view_component: README.md

end
```

### Template Inheritance

> Components that subclass another component automatically inherit their parent's

Reviewer: ```suggestion
Components that subclass another component inherit the parent component's
```

template if they don't define their own template.

```ruby

> # If `my_link_component.html.erb` is not defined so the component will fall back

Reviewer: ```suggestion
# If `my_link_component.html.erb` is not defined, the component will use
```
